### ISSUE TITLE
I cannot find a option to disable auto-update of STL-git...

### ISSUE THREAD

**Issue by TigerBeanst** [sonic2kk/steamtinkerlaunch#961]
## System Information
- SteamTinkerLaunch version: the latest git version
- Distribution: SteamOS 3.5 of Steam Deck
- Installation Method: ProtonUp-Qt (from Discover Store)

## Issue Description

I have enabled the "Advanced Mode" option in ProtonUp-Qt and make sure I am using the latest git version, not v12.12.

I found there has a word in wiki:

> On Steam Deck, SteamTinkerLaunch will automatically update to the latest Git, **but you are free to enable this option** and install SteamTinkerLaunch Git anyway.

I cannot find the option in STL setting( I launch it use `steamtinkerlaunch settings` ), is there anyway to disable the auto-update?

This is really annoying. Every time I launch something related to STL, I have to go through a check for updates that takes several seconds to several tens of seconds.

Is it because I didn't find the option correctly? I wish I could only update the STL when I manually use ProtonUp-Qt.

I have already tried to completely uninstall STL (delete `/dev/shm/steamtinkerlaunch` `~/.config/steamtinkerlaunch`, `~/stl`) then re-install it from ProtonUp-QT, still.

**Comment by TigerBeanst** [comment]
Oh, I got it in #859 , This seems like a feature that has not been developed yet.

**Comment by sonic2kk** [comment]
Historically this was intentional, because SteamOS support was really bad. But I don't see why it would be annoying, since it's only pulling < 1mb in most cases.

The section you have quoted is referring to the ProtonUp-Qt Advanced Mode, sorry if you misunderstood.

Also, to make extra clear, this issue is only on SteamOS, which is by far the worst place to use STL. SteamTinkerLaunch is for Linux Desktop users first and foremost and always will be, make sure to note such in your issue title, because SteamOS support is a massively low priority. The general userbase of SteamOS even killed my motivation for this project for a long time.

If you'd like to change this feel free to, but I personally most probably won't, because I have little interest in developing and also then testing such a feature for a device I'm not keen on, with a userbase that is not made up primarily of Linux enthusiasts, and it's a feature I won't even use because I don't use SteamTinkerLaunch on my Steam Deck.

So to answer your question:

> is there anyway to disable the auto-update?

Not until someone implements it, and I have far higher priorities than tinkering on a device that is not made for tinkering.
